src/components/dialogs/delete-dialog.js

```diff
@@ -21,7 +21,7 @@ import React from 'react';
  * @param {EventListener} onClose Event to close the dialog
  * @param {EventListener} onClick Event to submit the deletion
  * @param {String} title Title of the dialog
- * @param {String} message Message of the dialog
+ * @param {Object} message Message of the dialog
```

Comment:
not ok. The deletion messages will mainly have context in this dialog not elsewhere.
I think we should pass an Array of elementsToDelete and then we must build messages here in this component from this prop.

Reply:
done